packages/components/inputs/localized-money-input/src/localized-money-input.tsx

```diff
@@ -65,7 +250,8 @@ const LocalizedInput = (props) => {
         `}
       >
         <MoneyInput
-          id={props.id}
+          menuShouldBlockScroll={undefined} //NB: had to add missing prop
+          id={props.id} //NB: Had to add missing prop - open for dicsussion
```

Comment:
Can you clarify what these comments mean?

Reply:
`menuShouldBlockScroll` prop seems to be mandatory in `MoneyInput` component and it was not used anywhere in `localized-money-input.` . I had to pass it as an undefined value. I guess we may have to make it optional on `MoneyInput`. What do you think?

Reply:
Yes, it should be optional in `MoneyInput`. It's a regression from the migration to TS.

<img width="429" alt="image" src="https://user-images.githubusercontent.com/1110551/155131291-ea61478c-e448-451f-820d-0a4b567d3120.png">